aea/cli_gui/__init__.py

```diff
@@ -0,0 +1,425 @@
+# -*- coding: utf-8 -*-
```

Comment:
Looks good. But if we could create a `aea/cli_gui/__main__.py` file with some relevant parts of `aea/cli_gui/__init__.py` would be the best. With that change, we could run `python -m aea.cli_gui`.

Indeed, if I execute that command now:
```
$ python -m aea.cli_gui
/home/marcofavorito/.virtualenvs/agents-aea-jzIedExH/bin/python: No module named aea.cli_gui.__main__; 'aea.cli_gui' is a package and cannot be directly executed
```

Reply:
Hi Marco, 

If I have the run function (or nay function) in __init__.py can I call them from __main__.py?  how do I do that? Do I have to import something into __main__.py

Reply:
In particular, we could move [these lines](https://github.com/fetchai/agents-aea/pull/223/files/45c9252b638a1ba5e4b8f344d44ab399d1846f90#diff-bb1b8b3aa7790d5967a26dded4a97fb4R387-R425) into `aea/cli_gui/__main__.py`